pkg/cmd/roachtest/tests/perturbation/backup.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package perturbation
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/testutils"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/stretchr/testify/require"
+)
+
+// backup will run a backup during the perturbation phase.
+type backup struct{}
+
+var _ perturbation = backup{}
+
+func (b backup) setup() variations {
+	return setup(b, math.Inf(1))
+}
+
+// TODO(baptist): Add variations for incremental backup.
+// TODO(baptist): Add variations for backup with encryption.
```

Comment:
encryption should not matter too much. revision history backups could lead to a more a expensive backup though

Reply:
Is revision history backup different than incremental backup?

Reply:
yeah, it's specified with the `revision_history` option. 

https://www.cockroachlabs.com/docs/stable/take-backups-with-revision-history-and-restore-from-a-point-in-time

Reply:
I added a TODO to later add the revision history as well. I'm hoping someone from DR picks up these tests and adds the options in the future, but for now it will be good to get the base test implemented.